Rivet/Functions/Get-RivetConfig.ps1

```diff
@@ -272,9 +272,15 @@ function Get-RivetConfig
 
     $order = Get-ConfigProperty -Name 'DatabaseOrder' -AsArray
     $pluginModules = Get-ConfigProperty -Name 'PluginModules' -AsArray
+    $hasDatabaseOrder = $false

```

Comment:
I would update `Get-RivetConfig` so it orders the databases first my file system, them by the order the in the `DatabaseOrder` setting. That way, no other part of the code has to worry about the concept of ordering, and can just iterate through the list of databases.